sdk/keyvault/azure-keyvault-keys/README.md

```diff
@@ -165,42 +182,68 @@ print(ec_key.key_type)
 ```
 
 ### Retrieve a Key
-`get_key` retrieves a key previously stored in the vault.
+[`get_key`](https://aka.ms/azsdk-python-keyvault-keys-get-key-ref) retrieves a key previously stored in the vault.
 ```python
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.keys import KeyClient
+
+credential = DefaultAzureCredential()
+
+key_client = KeyClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
 key = key_client.get_key("key-name")
 print(key.name)
 ```
 
 ### Update an existing Key
-`update_key` updates a key previously stored in the Key Vault.
+[`update_key_properties`](https://aka.ms/azsdk-python-keyvault-keys-update-key-ref) updates the properties of a key previously stored in the Key Vault.
 ```python
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.keys import KeyClient
+
+credential = DefaultAzureCredential()
+
+key_client = KeyClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+
 # Clients may specify additional application-specific metadata in the form of tags.
 tags = {"foo": "updated tag"}
 
-updated_key_properties = key_client.update_key_properties("key-name", tags=tags)
+updated_key = key_client.update_key_properties("key-name", tags=tags)
```

Comment:
Everyone who read this thought we were trying to teach them that updateable properties are all tags. So I'm thinking we'd better show e.g. disabling a key instead.

Reply:
switched the keys one to just toggle enable, while the secrets one both disables the secret and updates content-type. Lmk what you think!